mmv1/products/compute/ForwardingRule.yaml

```diff
@@ -46,152 +46,138 @@ async: !ruby/object:Api::OpAsync
-      rigm_name: "website-rigm"
-      network_name: "website-net"
-      fw_name: "website-fw"
+      forwarding_rule_name: 'website-forwarding-rule'
+      region_target_http_proxy_name: 'website-proxy'
+      region_url_map_name: 'website-map'
+      region_backend_service_name: 'website-backend'
+      region_health_check_name: 'website-hc'
+      rigm_name: 'website-rigm'
+      network_name: 'website-net'
+      fw_name: 'website-fw'
     ignore_read_extra:
-      - "port_range"
-      - "target"
+      - 'port_range'
+      - 'target'
   - !ruby/object:Provider::Terraform::Examples
-    name: "forwarding_rule_regional_http_xlb"
+    name: 'forwarding_rule_regional_http_xlb'
     min_version: 'beta'
-    primary_resource_id: "default"
+    primary_resource_id: 'default'
     vars:
-      forwarding_rule_name: "website-forwarding-rule"
-      region_target_http_proxy_name: "website-proxy"
-      region_url_map_name: "website-map"
-      region_backend_service_name: "website-backend"
-      region_health_check_name: "website-hc"
-      rigm_name: "website-rigm"
-      network_name: "website-net"
-      fw_name: "website-fw"
-      ip_name: "website-ip"
+      forwarding_rule_name: 'website-forwarding-rule'
+      region_target_http_proxy_name: 'website-proxy'
+      region_url_map_name: 'website-map'
+      region_backend_service_name: 'website-backend'
+      region_health_check_name: 'website-hc'
+      rigm_name: 'website-rigm'
+      network_name: 'website-net'
+      fw_name: 'website-fw'
+      ip_name: 'website-ip'
     ignore_read_extra:
-      - "port_range"
-      - "target"
-      - "ip_address"
+      - 'port_range'
+      - 'target'
+      - 'ip_address'
   - !ruby/object:Provider::Terraform::Examples
-    name: "forwarding_rule_vpc_psc"
+    name: 'forwarding_rule_vpc_psc'
     min_version: 'beta'
-    primary_resource_id: "default"
+    primary_resource_id: 'default'
     vars:
-      forwarding_rule_name: "psc-endpoint"
-      consumer_network_name: "consumer-net"
-      ip_name: "website-ip"
-      producer_network_name: "producer-net"
-      producer_psc_network_name: "producer-psc-net"
-      service_attachment_name: "producer-service"
-      producer_forwarding_rule_name: "producer-forwarding-rule"
-      producer_backend_name: "producer-service-backend"
-      producer_healthcheck_name: "producer-service-health-check"
+      forwarding_rule_name: 'psc-endpoint'
+      consumer_network_name: 'consumer-net'
+      ip_name: 'website-ip'
+      producer_network_name: 'producer-net'
+      producer_psc_network_name: 'producer-psc-net'
+      service_attachment_name: 'producer-service'
+      producer_forwarding_rule_name: 'producer-forwarding-rule'
+      producer_backend_name: 'producer-service-backend'
+      producer_healthcheck_name: 'producer-service-health-check'
     ignore_read_extra:
-      - "port_range"
-      - "target"
-      - "ip_address"
-  - !ruby/object:Provider::Terraform::Examples
```

Comment:
For some reason this example is getting removed as part of this change - is that intentional?

Reply:
No, should be fixed now.